Formula/nss.rb

```diff
@@ -8,6 +8,9 @@ class Nss < Formula
   livecheck do
     url "https://ftp.mozilla.org/pub/security/nss/releases/"
     regex(%r{href=.*?NSS[._-]v?(\d+(?:[._]\d+)+)[._-]RTM/?["' >]}i)
+    strategy :pagematch do |page, regex|
```

Comment:
```suggestion
    strategy :page_match do |page, regex|
```
While `:pagematch` seems to work, shouldn't this actually be `:page_match`, at least for consistency with other formulae?